### ISSUE TITLE
fastlane requires a minimum version of Xcode 7.0

### ISSUE THREAD

**Issue by lkraider** [fastlane/fastlane#9695]
### New Issue Checklist

- [ X ] Updated fastlane to the latest version
- [ X ] I have read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)

### Issue Description

Fastlane refuses to run any command when Xcode is set to an older version.

We have some CIs that build on old Xcode versions that use fastlane sigh to download the provisioning profiles. Since upgrading from fastlane 2.37 to 2.45 fastlane will not run anymore and complain about Xcode version.

##### Complete output when running fastlane, including the stack trace and command used
> You can use: `--capture_output` as the last commandline argument to get that collected for you

```
DEVELOPER_DIR=/Applications/Xcode-4.5.2.app/Contents/Developer/

21:35:46 + sigh --adhoc --username XXX --team_id XXX --app_identifier 'XXX' --skip_install

21:35:48 Fastlane update available
21:35:48 Please update fastlane by downloading an updated bundle from
21:35:48 https://download.fastlane.tools

21:35:51 [WARNING] You are calling sigh directly. Usage of the tool name without the `fastlane` prefix is deprecated in fastlane 2.0
21:35:51 Please update your scripts to use `fastlane sigh --adhoc --username XXX --team_id XXX --app_identifier XXX --skip_install` instead.

21:35:53 
21:35:53 [!] fastlane requires a minimum version of Xcode 7.0, please upgrade and make sure to use `sudo xcode-select -s /Applications/Xcode.app`
```

### Environment

Please run `fastlane env` and copy the output below. This will help us help you :+1:
If you used `--capture_output` option please remove this block - as it is already included there.

```
$ fastlane env
Fastlane update available
Please update fastlane by downloading an updated bundle from
https://download.fastlane.tools

[!] fastlane requires a minimum version of Xcode 7.0, please upgrade and make sure to use `sudo xcode-select -s /Applications/Xcode.app`
```

**Comment by lkraider** [comment]
Breakage was introduced by https://github.com/fastlane/fastlane/pull/9609

**Comment by lkraider** [comment]
Editing `MINIMUM_XCODE_RELEASE = "4.0"` fixes everything in:
`~/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.45.0/fastlane/lib/fastlane/version.rb`

**Comment by KrauseFx** [comment]
Hey @lkraider, thanks for reporting this issue. Is there a chance to upgrade Xcode to Xcode 7? We can't guarantee working with pre Xcode 7 releases, especially around the installed version of Ruby and OpenSSL, in combination with updated `xcodebuild` commands.

**Comment by lkraider** [comment]
Indeed this machine has many Xcode versions installed (it's a build system), I just need the commands that don't require Xcode - like the ones used for Android - to work. Not looking to run gym for example, my need is mostly to automate itunes/member center.

**Comment by farwayer** [comment]
One more problem with this change is `No such file or directory - xcode-select` warning on linux host. Maybe it is better to check version only for XCode specific tasks?

**Comment by GokhanArik** [comment]
I'm using fastlane for Android builds only and it is asking me to upgrade XCode also. Any workaround?

**Comment by mpirri** [comment]
@GokhanArik Thanks for letting us know about this. Out of curiosity, do you have any version of Xcode installed?

Also, can you share a stack trace from the error message? It sounds like there might be a bug here that we'd love to resolve.

Thanks so much!  🚀

**Comment by GokhanArik** [comment]
Hi @mpirri 

I did investigate a little bit, the problem is that Fastlane is using `xcode-select -p`. I have XCode installed because I'm using developer tools, especially difftool.

So my path is `/Applications/Xcode.app/Contents/Developer`. 

I tried changing it with `sudo xcode-select -s /`, and it works. The path becomes `/Library/Developer/CommandLineTools`. However, I don't want to change that, otherwise, I won't be able to use developer tools. 

I don't have full message now but it comes from here, it just said I needed version 7.0 I believe: https://github.com/fastlane/fastlane/blob/de0942b1d1202505bbc80c549b650724ea8c318c/fastlane_core/lib/fastlane_core/ui/fastlane_runner.rb#L60
